.travis.yml

```diff
@@ -11,6 +11,7 @@ env:
     global:
         - SYMFONY_PHPUNIT_DIR="$HOME/symfony-bridge/.phpunit"
         - SYMFONY_PHPUNIT_REMOVE="symfony/yaml" # keep Prophecy
+        - SYMFONY_DEPRECATIONS_HELPER=2 # we need to allow our own deprecations
```

Comment:
we should not do that. We should mark the test triggering them as legacy instead.